src/test/java/com/regnosys/testing/pipeline/PipelineTestHelper.java

```diff
@@ -164,41 +140,6 @@ PipelineTreeConfig createTreeConfigWithoutStarting() {
                 .add(middleClass(), TransformType.PROJECTION, endClass());
     }
 
-    PipelineTreeConfig createTreeConfigForFilter() {
```

Comment:
Quite a few tests deleted, are they not needed?

Reply:
We had three different ways to filter earlier and the test was testing each of them with three reports within one pipeline chain. Now there is just one way to filter and we have covered all those in the one pipeline chain with one report. happy to chat if you like